plugins/hls-pragmas-plugin/test/Main.hs

```diff
@@ -15,8 +15,13 @@ import           Test.Hls.Util           (onlyWorkForGhcVersions)
 main :: IO ()
 main = defaultTestRunner tests
 
+-- FIXME: Combine these plugins into a single one
+
 pragmasPlugin :: PluginTestDescriptor ()
```

Comment:
I dont think this fixme is necessary :)

Reply:
Not sure why CI is succeeding.
The pragma plugin test is failing locally for me, because the "disable warning" code actions are missing.

I don't think combining the plugins is the solution though - I should probably add separate tests for the "disable warnings" descriptor.